contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -551,11 +551,26 @@ def _execute(self, preset_name: str):
         status_text = presets_panel.get_item_at(preset_row, 0).texts()[0]
 
         if 'No' in status_text:
+            logging.info("About to click File->OpenPreset...")
             app_menu = self.suite.top_window().descendants(control_type="MenuBar")[1]
             app_menu.item_by_path("File->Open Preset...").click_input()
-            dialog = self.suite.top_window().child_window(title_re="Select a file*")
-            dialog.FileNameEdit.type_keys(preset_name)
-            dialog.FileNameEdit.type_keys('{DOWN}{ENTER}')
+
+            def get_open_file_dialog():
```

Comment:
Can you move this outside of this method?

Reply:
No.. I would like to keep it in, because moving it out is not super trivial. I thought about deduping code with similar functionality from the end of the file. But its quite some work and only little benefit